tutorial.ipynb

```diff
@@ -288,6 +280,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Also, note the the lack of smoothness in the voltage curve. This is limited by the save_freq from sim_cfg. There is a point on the graph every 20 seconds, because that is the frequency at which we save the results. Decreasing the save frequency will result in a cleaner curve.\n",
```

Comment:
```suggestion
    "Also, note the the lack of smoothness in the voltage curve. This is limited by the save_freq from sim_cfg. There is a point on the graph every 20 seconds, because that is the frequency at which we save the results. Decreasing the save frequency will result in a smoother curve.\n",
```
Just a suggestion